decidim-core/lib/tasks/decidim_tasks.rake

```diff
@@ -81,4 +81,9 @@ namespace :decidim do
       log.info "RightToBeForgotten: --------------- END #{Time.current}"
     end
   end
+
+  desc "Check and notify users to update her newsletter notifications settings"
+  task check_users_newsletter_opt_in: :environment do
+    Decidim::User.where(newsletter_notifications: true).each(&:newsletter_opt_in_notify)
```

Comment:
* Shouldn't we filter those who have already opt in?
* Use `find_each` instead of `each`